ntoskrnl/ps/process.c

```diff
@@ -829,6 +875,7 @@ PspCreateProcess(OUT PHANDLE ProcessHandle,
         Process->GrantedAccess |= (PROCESS_VM_OPERATION |
                                    PROCESS_VM_READ |
                                    PROCESS_VM_WRITE |
+                                   PROCESS_QUERY_LIMITED_INFORMATION |
```

Comment:
This is not correct. And it should not be necessary -- PROCESS_QUERY_INFORMATION implies PROCESS_QUERY_LIMITED_INFORMATION.